docker/prod/prod.template.yaml

```diff
@@ -246,6 +252,12 @@ spec:
               port: 9000
             timeoutSeconds: 5
           env:
+            - name: KUBERNETES_DEPLOYMENT_NAME
+              value: mediawiki-prod
```

Comment:
Same question as https://github.com/Wikia/app/pull/15648/files#r204779888 but with `mediawiki-tasks`

Reply:
Changed in 41e6da58164